content/pages/about/technology-overview.md

```diff
@@ -108,26 +108,26 @@ _Note: Many of the following implementations support the older Google Talk proto
 
 __Clients__
 
-- [Coccinella](http://coccinella.im/)
-- [Gajim](http://www.gajim.org/)
-- [Jitsi (formerly named SIP Communicator)](http://jitsi.org/)
+- [Coccinella](https://coccinella.im/)
+- [Gajim](https://www.gajim.org/)
+- [Jitsi (formerly named SIP Communicator)](https://jitsi.org/)
 - [Pandion](http://pandion.im/)
-- [Pidgin (formerly named Gaim)](http://pidgin.im/)
+- [Pidgin (formerly named Gaim)](https://pidgin.im/)
 - [Psi](http://psi-im.org/)
-- [Telepathy](http://telepathy.freedesktop.org/)
+- [Telepathy](https://telepathy.freedesktop.org/)
 - [Yate](http://yate.null.ro/)
 
 __Libraries__
 
-- [libjingle](http://code.google.com/apis/talk/) (C/C++)
-- [Smack](http://www.igniterealtime.org/projects/smack/) (Java)
-- [Telepathy Gabble](http://telepathy.freedesktop.org/) (C)
+- [libjingle](https://code.google.com/apis/talk/) (C/C++)
```

Comment:
This link is a 404, updating it to https isn't really useful. I suppose it doesn't hurt either, but maybe this should be fixed or removed instead.